services/src/main/resources/locale/jitsi/Jitsi_en.xml

```diff
@@ -2,7 +2,7 @@
 <bundle>
   <UICallButton>
     <label>
-      <jitsi>Call</jitsi>
+      <jitsi>Jitsi Call</jitsi>
```

Comment:
this should be modified on Crowdin instead @hghorbel

Reply:
Done : https://crowdin.com/translate/exo-platform/36702/en-en?filter=basic&value=3